recipes/sirius-ms/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "sirius" %}
+{% set version = "5.6.4" %}
+{% set commit = "cbc32a4754e0595baed0575f86c95bb4198098f4" %}
+
+package:
+  name: {{ name|lower }}-ms
+  version: {{ version }}
+
+source:
+  - url: https://github.com/boecker-lab/sirius/archive/{{ commit }}.zip
+    sha256: 5e7527ccff6e3b5c0877b93f497adc5d74528aca3caab369add1aabdca157f3f
+
+build:
+  number: 0
+  binary_relocation: False
+  ignore_prefix_files: True
+  detect_binary_files_with_prefix: False
+
+requirements:
+  build:
+    - openjdk
+  run:
+    - openjdk
+    - coin-or-cbc  # [not win]
+
+test:
+  requires:
+    - wget  # [not win]
+
+about:
+  home: https://bio.informatik.uni-jena.de/software/sirius/
+  license: AGPL-3.0-only
+  license_file:
+    - COPYING.txt
+    - LICENSE.txt
```

Comment:
Because you are redistributing many of the java libraries that your package depends on, you need to include their licenses in this section. I think there is a gradle-plugin/tool that can collect the licenses automatically into a single file, but I don't know what it's called because I do not use java.

i.e. license information for amqp, annotations, api-common, base64, batik, bcprov, beam, blob-storage, canopus, ...

Reply:
The gradle build now creates a `licenses.md` file that lists the license information of each package included in SIRIUS.